app/helpers/height_helpers.py

```diff
@@ -0,0 +1,10 @@
+from app.helpers.helpers import filter_altitude
+
+
+def get_height(spatial_reference, easting, northing, georaster_utils=None):
```

Comment:
Consider setting the `georaster_utils` as positional argument instead of key arg with default value to None if it is required. The assert below can then be removed.

Reply:
isn't that something only supported in python 3.8? Do you mean `def get_height(spatial_reference, easting, northing, georaster_utils, /):` ?

Reply:
No simply `def get_height(spatial_reference, easting, northing, georaster_utils):` I don't see use case to use keyword arg here ?

Reply:
it was to make sure I received a valid instance (so that I wouldn't forget to add it somewhere in my code) but yeah, now it's done we could get rid of this safety